frontends/web/src/routes/base/base.tsx

```diff
@@ -0,0 +1,111 @@
+/**
```

Comment:
i prefer more explicit file/object names for the base, as `base` is a very generic and overloaded workd. `bitboxbase.tsx`, `BitBoxBase`, etc. will help.